precompiles/parachain-staking/StakingInterface.sol

```diff
@@ -0,0 +1,305 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
We have all of the other contracts without `Interface` in their filenames. Can we rename this file `Staking.sol`, please?

Reply:
Sure, I will use `ParachainStaking` (which is the name of the pallet)